### ISSUE TITLE
Make use of World Files for Images etc.

### ISSUE THREAD

**Issue by bethanywallace** [Leaflet/Leaflet#8876]
### Checklist

- [X] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

It becomes quite a manual process having to set image coordinates to display on a world map. Could a feature be implemented in which we can point the code towards an images related world file and it can determine its location from there?

### Suggested solution

Extract the data in the world file and use to calculate the image coordinates

**Comment by IvanSanchez** [comment]
I think this is out of scope of core leaflet.

Leaflet only implements conformal raster display in `L.ImageOverlay` (i.e. "same projection as the map, and with zero rotation relative to the horizontal axis of the screen").

The two plugins I made a while ago (https://github.com/IvanSanchez/Leaflet.ImageOverlay.Rotated and https://gitlab.com/IvanSanchez/Leaflet.ImageOverlay.Arrugator) each have their issues. ImageOverlay.Rotated needs the corners to be given as `L.LatLng`s, and ImageOverlay.Arrugator needs `proj4` data to work.

The main issue is that a worldfile contains coordinates **relative to the CRS of the raster**, which is a third bit of information that has to be fed to the algorithm (besides the image size and the worldfile). I don't think it's worth it to do in ImageOverlay.Rotated, and it becomes trivial in ImageOverlay.Arrugator, *but only after the projection data has been set*.

I don't know if any other plugins at https://leafletjs.com/plugins#tileimage-display do worldfiles.
